app/domain/authentication/authn_jwt/vendor_configurations/configuration_jwt_generic_vendor.rb

```diff
@@ -80,22 +91,10 @@ def fetch_signing_key
           )
         end
 
-        def authentication_parameters_class
-          @authentication_parameters_class ||= Authentication::AuthnJwt::AuthenticationParameters
-        end
-
         def extract_token_from_credentials
           @extract_token_from_credentials ||= Authentication::AuthnJwt::InputValidation::ExtractTokenFromCredentials.new
         end
```

Comment:
this should be moved to the method signature

Reply:
Done